Makefile

```diff
@@ -9,18 +9,34 @@ IMAGE_NAME := node-feature-discovery
 IMAGE_TAG_NAME := $(VERSION)
 IMAGE_REPO := $(IMAGE_REGISTRY)/$(IMAGE_NAME)
 IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
+K8S_NAMESPACE := kube-system
 
+yaml_templates := $(wildcard *.yaml.template)
+yaml_instances := $(patsubst %.yaml.template,%.yaml,$(yaml_templates))
 
 all: image
 
-image:
+image: $(yaml_instances)
```

Comment:
Couldn't/shouldn't `image` depend simply on the `yamls` target?

Reply:
Yup, it should. Thanks!